content/Proposals/Done/202106-proposals-process.md

```diff
@@ -0,0 +1,112 @@
+      3. Fill sections
+      4. Announce it on the team mailing list and Slack channel
+      5. Address comments / concerns 6 Define what "done" means for this proposal, i.e. what is the purpose of this design document:
+      * Knowledge sharing / Brain dump: This kind of document may not need a thorough review or any official approval
+      * Long term vision and Execution & Implementation: If approved (with LGTM comments, or in an approved section) by a majority of the team and no major concerns consider it approved. NOTE: The same applies to rejected proposals.
+      1. If the document has no more offline comments and no consensus was reached, schedule a meeting with interested parties.
+      2. When the document changes status, move it to the appropriate status folder in the design docs directory of the team folder. If an approved proposal concerns a component with its own directory, e.g. Telemeter, then create a shortcut to the proposal document in the component-specific directory. This helps us find design documents by topic and by status.
+
+It served us well, but it had the following issues (really similar to ones stated in [handbook proposal](../Accepted/202106-handbook.md#pitfalls-of-the-current-solution)):
+
+* Even if our Google Design docs organized in our team drive, those Google documents are not easily discoverable.
+* Existing Google doc-based documents are hard to consume. The formatting is widely different. Naming is inconsistent.
+* Document creation is rarely actionable. There is no review process, so the effort of creating a relevant document might be wasted, as the document is lost. This also leads to docs being in the half-completed state, demotivating readers to look at it.
+* It's hard to track previous discussions around proposals, who approved them (e.g. proposals).
+* It's not public, and it's hard to share good proposals with other external and internal teams.
+
+## Goals
+
+Goals and use cases for the solution as proposed in [How](#how):
+
+* Allow easy collaboration and decision making on design ideas.
+* Have a consistent design style that is readable and understandable.
+* Ensure design docs are discoverable for better awareness and knowledge sharing about past decisions.
+* Define a clear review and approval process.
+
+## Non-Goals
+
+* Define process for other documents (see [handbook proposal](../Accepted/202106-handbook.md#pitfalls-of-the-current-solution))
+
+## How
+
+We want to propose an improved, official proposal process for Monitoring Group that clearly states *when, where and how* to create proposal/enhancement/design documents.
+
+Everything starts with a problem statement. It might be a missing functionality, confusing existing functionality or broken one. It might be an annoying process, performance or security issue (or potential one).
+
+### What Process to Follow?
```

Comment:
I would rename the subtitle to something similar to `Where to propose changes/Where to submit proposals`. I had to read the entire document 2-3 times to understand the difference between `What Process to Follow` and `Handbook Proposal Process`.

Reply:
Yessss